apps/web/next.config.js

```diff
@@ -278,6 +278,10 @@ const nextConfig = {
           source: "/cancel/:path*",
           destination: "/booking/:path*",
         },
+        {
+          source: "/routing-forms/:slug*",
```

Comment:
Routing Forms is now system app. So, allow accessing it directly. This also allows us to never have to move Routing Forms app outside AppStore unless needed for some other reason.